regress/core/tickets.sql

```diff
@@ -1575,3 +1575,7 @@ WHERE ST_DFullyWithin(
     extent,
     1700
 );
+
+SELECT '#####', ST_AsText(ST_AddPoint(
```

Comment:
We have the same problem with `ST_SetPoint`:

SELECT ST_AsText(ST_SetPoint('LINESTRING(1 1, 2 2, 3 3)', 1, 'POINT EMPTY'));
                           st_astext                            
----------------------------------------------------------------
 LINESTRING(1 1,3.3425537346456e-310 3.34255373464587e-310,3 3)
(1 ligne)